src/stdlib_experimental_io.fypp

```diff
@@ -157,20 +165,25 @@ contains
 
 
   integer function open(filename, mode, iostat) result(u)
-    ! Open a file
-    !
-    ! To open a file to read:
-    !
-    ! u = open("somefile.txt")        # The default `mode` is "rt"
-    ! u = open("somefile.txt", "r")
-    !
-    ! To open a file to write:
-    !
-    ! u = open("somefile.txt", "w")
-
-    ! To append to the end of the file if it exists:
-    !
-    ! u = open("somefile.txt", "a")
+    !! Opens a file
+    !!
+    !! __Read the [specification here](../page/specs/stdlib_experimental_io.html#description_1)__
+    !!
+    !!##### Behavior
+    !!
+    !!
+    !! To open a file to read:
+    !!
+    !! u = open("somefile.txt")        # The default `mode` is "rt"
+    !! u = open("somefile.txt", "r")
```

Comment:
Just wondering, why is `#` used for a comment here? You should be able to use the normal Fortran comment character, `!`. Also, if you don't want to embed FORD links/crossrefs,  then I would recommend putting this in a pre-formatted code block. This will turn on syntax highlighting (using pygments) in the generated documentation.

e.g.;
 ```suggestion
    !!``` fortran
    !! u = open("somefile.txt")        # The default `mode` is "rt"
    !! u = open("somefile.txt", "r")
    !!```
```